### PR TITLE
CorpusViewer: Break long urls

### PR DESCRIPTION
**Pull request by nikicc**
##### Issue

Long URLs cause the horizontal scroll bar in CorpusViewer.

##### Description of changes
Break more.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by nikicc** [comment]
@lanzagar this quick and dirty fix works for me.

File: orangecontrib/text/widgets/owcorpusviewer.py [hunk 1]
```diff
@@ -245,6 +245,26 @@ def show_docs(self):
...
+            word-break: break-all;
+            /* Instead use this non-standard one: */
+            word-break: break-word;
+        
+            /* Adds a hyphen where the word breaks, if supported (No Blink) */
+            -ms-hyphens: auto;
+            -moz-hyphens: auto;
+            -webkit-hyphens: auto;
+            hyphens: auto;
+        }}
```
Comment (kernc) [inline orangecontrib/text/widgets/owcorpusviewer.py]: This can be trimmed to three lines: overflow-wrap, word-wrap, word-break: break-all; hyphen.

**Comment by ajdapretnar** [comment]
Perhaps this is something we can update and merge?

**Comment by kernc** [comment]
Sure.

**Pull request merged**